### PR TITLE
CNV-9457 - RN - CDI config params moved to CDI object

### PR DESCRIPTION
**Pull request by aburdenthehand**
RN added for change to CDI configuration

### CODE REVIEW

**Review by aglitke — APPROVED** [review]
/lgtm

File: virt/virt-2-6-release-notes.adoc [hunk 1]
```diff
@@ -74,8 +74,7 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-9339: When deploying from the GUI, the CR for the HCO is now created during subscription
 
-//CNV-9457: CDI configuration parameters have moved from the CDIConfig object to the CDI CR
-
+* The Container Data Importer (CDI) configuration parameters have moved from the `CDIConfig` object to the `CDI` object. All changes to the CDI configuration must now be made in the `CDI` object.
```
[the remark this answers is not in the document]
Comment (aburdenthehand) [reply virt/virt-2-6-release-notes.adoc]: Makes sense. Updated as suggested.

[the remark this answers is not in the document]
Comment (akalenyu) [reply virt/virt-2-6-release-notes.adoc]: Thanks, otherwise LGTM

File: virt/virt-2-6-release-notes.adoc [hunk 2]
```diff
@@ -77,11 +77,11 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //CNV-9339: When deploying from the GUI, the CR for the HCO is now created during subscription
 * The procedure for xref:../virt/install/installing-virt-web.adoc#installing-virt-web[installing {VirtProductName} in the web console] has been streamlined. You can now create the `HyperConverged` custom resource immediately after installing the {VirtProductName} Operator by clicking *Create HyperConverged*.
 
-//CNV-9457: CDI configuration parameters have moved from the CDIConfig object to the CDI CR
-
 //CNV-10153: BareMetalPlatform removed from HCO CR
 * Previously, there was a `spec.BareMetalPlatform` field in the `HyperConverged` object. This field has been removed.
 
+* The Container Data Importer (CDI) configuration parameters have moved from the `CDIConfig` object to the `CDI` object. All changes to the CDI configuration must now be made in the `spec.config` field of the `CDI` object.
```
Comment (sjhala-ccs) [inline virt/virt-2-6-release-notes.adoc:83]: s/Container/Containerized

**Review by sjhala-ccs — COMMENTED** [review]
One nit, otherwise LGTM

**Review by jeana-redhat — APPROVED** [review]
approving exception

**Pull request merged**